RadixWallet/Features/AccountRecoveryScan/Coordinator/AccountRecoveryScanCoordinator.swift

```diff
@@ -158,12 +158,10 @@ public struct AccountRecoveryScanCoordinator: Sendable, FeatureReducer {
 		case let .createProfile(privateHD):
 			let recoveredAccountAndBDFS = AccountsRecoveredFromScanningUsingMnemonic(
 				accounts: sortedAccounts,
-				deviceFactorSource: privateHD.factorSource
+				factorSource: privateHD
 			)
 			return .run { send in
 				let result = await TaskResult<EqVoid> {
-					try secureStorageClient.saveMnemonicForFactorSource(privateHD)
```

Comment:
Mnemonic will be forwarded to Sargon, and then Sargon will save it to secure storage.